natural-language-processing/ner.md

```diff
@@ -1,26 +1,91 @@
+**B-** Another method in this category is the Conditional Random Field (CRF) model. It is a probabilistic graphical model that can be used to model sequential data such as labels of words in a sentence. The CRF model can catch the features of the present and past names in an arrangement however it can’t comprehend the setting of the forward labels; this shortcoming plus the extra feature engineering involved with training a CRF model makes it less appealing to be adopted by the industry.  
+
+
+**2.3 Deep Learning Approaches:**  ⭐
+
+Before discussing details about Deep Learning approaches (state-of-the-art) to NER, we need to analyze proper and clear metrics to evaluate the performance of our models. Generally, we use accuracy while training a neural network in different epochs as an evaluation metric. However, in the case of NER, we might be dealing with important financial, medical, or legal documents and accurate identification of named entities in those documents determines the success of the model. In other words, false positives and false negatives have a business cost in a NER task. Therefore, our principal metric to evaluate our models will be the F1 score because we need a balance between precision and recall.  
+
+**3 Evaluation Metrics of NER**  
+
+**3.1 CoNLL: Computational Natural Language Learning**  
+
+“Precision is the percentage of named entities found by the learning model that is correct. Recall is the percentage of named entities present in the corpus that are found by the model. A named entity is correct only if it is an exact match of the corresponding entity in the data file.”  
+The Language-Independent Named Entity Recognition task introduced at CoNLL-2003 measures the performance of the systems in terms of precision, recall, and f1-score.  
+
+
+**3.2 Automatic Content Extraction (ACE)**  
+
+The ACE challenges use a more complex evaluation metric which includes a weighting schema, Check References for deeper understanding.
+Replicating experiments and baselines from ACE are a little complex since all the datasets and results are not open and free, so I guess this challenge results and experiments will fade away with time.  
+
+**3.4 Message Understanding Conference (MUC)**  
+
+The MUC-5 Scoring System is evaluation software that aligns and scores the templates produced by the information extraction systems under evaluation in comparison to an "answer key" created by humans . The Scoring System produces comprehensive summary reports showing the overall scores for the templates in the test set ; these may be supplemented by detailed score reports showing scores for each template individually. To understand the scoring categories, see [this](https://www.aclweb.org/anthology/M93-1007.pdf)
+
+Below is a summary table with meaning of each scoring category!
+![scoring](https://github.com/UmaGunturi/awesome-ai-ml-dl/blob/master/natural-language-processing/formulae/scoring.png)
```

Comment:
Url is still pointing to https://github.com/UmaGunturi -  this is a simple change I think

Reply:
Done!